packages/lib/src/ComplexVisual.js

```diff
@@ -622,7 +623,7 @@ class ComplexVisual extends Visual {
             }
 
             if (DEBUG && !errorOccured) {
-              logger.debug(`Triangles count: ${gfxutils.countTriangles(repr.geo)}`);
+              logger.debug(`Triangles count: ${meshutils.countTriangles(repr.geo)}`);
```

Comment:
It would be better to import only the function countTriangles from meshutils as we use ony it.